utils/measure/measure/const.py

```diff
@@ -21,3 +21,14 @@ class MeasureType(str, Enum):
     RECORDER = "Recorder"
     AVERAGE = "Average"
     CHARGING = "Charging device"
+
+
+class Trend(StrEnum):
+    INCREASING = "increasing"
+    DECREASING = "decreasing"
+    STEADY = "steady"
+
+
+dummy_load_measurement_count = 20
+dummy_load_measurements_duration = 30
```

Comment:
Btw constants must be uppercased by python conventions.

Reply:
Weird that Ruff did not complain about these